src/config/XMLConfig.h

```diff
@@ -300,7 +291,6 @@ class XMLConfigParser
 		return set_attribute(name, value.GetValue());
 	}
 	
-	template <>
```

Comment:
?

Reply:
Fixes the need for -fdelayed-template-parsing flag in clang and make it work on gcc